### DISCUSSION TITLE
May I intercept messages sent to a private group in where the bot isnt a participant?

### DISCUSSION THREAD

**Discussion by carneiroDotDev** [telegraf/telegraf#1326] [Q&A]
Hey guys,

First of all, congrats for the project! I spent the weekend playing with it and 
the documentation is just fantastic. 🎉

Unfortunately, I think it doesn't fit my final need which is to capture all the messages 
sent in a private group and log them in a group of mine since some of them are deleted 
on the original group. I am not an admin in the private group. If I understood well, the 
Telegraf offers a `forwardMessage` method but the bot must be also a participant 
of the original group, right? 

On the other hand I still hope to use the Telegraf for my side project because I saw it offers 
all the mothods available on the Telegram Api via `bot.telegram`. My question is then if the 
Telegraf is enough to manipulate messages from a private group which the bot is not in? 
or should I look for Telethon or something else? 

Thanks in advance and congrats once more,
Luiz

**Comment by irokhes** [comment]
Hi @carneiroDotDev ! did you manage to solve this problem? I got a similar situation
